portmaster/install/linux.md

```diff
@@ -179,11 +177,31 @@ WantedBy=multi-user.target
 
 Finally, reload the systemd daemon and enable/start the Portmaster:
 
-```
+```sh
 sudo systemctl daemon-reload
 sudo systemctl enable --now portmaster
 ```
 
+__6bis.__ Start it on boot Runit (systems)
+
+In order to get the Portmaster Core Service to run automatically at boot, you need to make a runit service by first creating a directory at `/usr/local/sv/portmaster/` (if there isn't any `/usr/local/sv/` directory just create with the 755 permission using `mkdir -p /usr/local/sv` ) with 755 permissions, then creating a `run` file at `/usr/local/sv/portmaster/run` with the same permissions as the `portmaster` folder we created. This file **must** contain the following:
+
+```sh
+#!/bin/sh
+exec /opt/safing/portmaster/portmaster-start core
+```
+
+**NOTE**: The portmaster-start script might be located elsewhere.
+
+Finally, enable and start the service:
+
+```sh
+sudo ln -s /usr/local/sv/portmaster /run/runit/service/portmaster
```

Comment:
Oh boy. This one's a headache.

#### For running machine
- Artix linux suggests `/run/runit/service/portmaster`
- Void linux suggests `/var/service/portmaster`


I think we should go with `/etc/runit/runsvdir/default/`